README.md

```diff
@@ -32,36 +35,54 @@ This is the new repository ([old one](https://github.com/pylhc/Beta-Beat.src)) o
 
 ## Functionality
 
-Scripts to be executed lie in `/omc3` directory. These include
-- hole_in_one.py to perform frequency on turn by turn BPM data and infer optics and more for a given accelerator
-- madx_wrapper.py to start a MADX with a file or string as input
-- model_creator.py to provide optics models required for optics analysis
-- plot_spectrum.py to generate plots from files generated by frequency analysis
-- run_kmod.py to analyse data from K-modulation and return measured optics functions
-- tbt_converter.py to convert different turn by turn datatypes to sdds and add noise 
+The main scripts to be executed lie in `/omc3` directory. These include
+- `hole_in_one.py` to perform frequency analysis on turn by turn BPM data and infer optics and more for a given accelerator
+- `madx_wrapper.py` to start a MADX with a file or string as input
+- `model_creator.py` to provide optics models required for optics analysis
+- `run_kmod.py` to analyse data from K-modulation and return measured optics functions
+- `tbt_converter.py` to convert different turn by turn datatypes to sdds and add noise 
+- `amplitude_detuning_analysis.py` to perform amp. det. analysis on optics data with tune correction
+
+
+Plotting Scripts can be found in `/omc3/plotting`
+- `plot_spectrum.py` to generate plots from files generated by frequency analysis
+- `plot_bbq.py` to generate plots from files generated by frequency analysis
+- `plot_amplitude_detuning.py` to generate plots from files generated by frequency analysis
+
 
 Some examples can be found in the test files.
 
 ### Changelog
 
-- Still building up to first release.
 
-### Implemented
+#### 2020-02-24
+
+- Added amplitude detuning analysis
+- Distinction between `BasicTests` and `Extended Tests`
 
-- Madx wrapper
-- Frequency Analysis of turn by turn
-- Optics measurement analysis scripts
-- Accelerator class
-- K-mod
+#### Before
 
+- Updated and moved main functionalities from python 2.7
+    - Madx wrapper
+    - Frequency Analysis of turn by turn
+    - Optics measurement analysis scripts
+    - Accelerator class and Model Creator
+    - K-mod
+    - Spectrum Plotting
+    - Turn-by-Turn Converter
+
+- `setup.py` and packaging functionality 
+- Automated CI
+    - Multiple versions of python
+    - Accuracy tests
+    - Unit tests
+    - Release automation
```

Comment:
Should we have a Changelog or not?

Reply:
Maybe automated from commit messages to CHANGELOG.md in future